api/src/main/java/com/messagebird/MessageBirdClient.java

```diff
@@ -1009,7 +1009,7 @@ ConversationWebhookList listConversationWebhooks(final int offset, final int lim
      *
      * @return List of webhooks.
      */
-    public ConversationWebhookList listConversationWebHooks() throws UnauthorizedException, GeneralException {
+    public ConversationWebhookList listConversationWebhooks() throws UnauthorizedException, GeneralException {
```

Comment:
Lets rename "listConversationWebhooks" method name as listConversationWebhookWithDefaultPaginationOptions

Reply:
Also, on line 1001 rename "listConversationWebhooks" method as listConversationWebhookWithSpecifiedPaginationOptions and make it public.

Reply:
Let’s create a separate PR for that, it’s not related to voice webhooks.

Reply:
okay, then can you also revert your modification about method name in this line. 
"listConversationWebhooks" should be reverted.

Reply:
This is a typo fix. Why should it be reverted?

Reply:
oh okay then. I thought this change is also unrelated to voice webhook.